server/src/test/java/org/spine3/testdata/Exemplum.java

```diff
@@ -52,14 +52,14 @@
 import static java.lang.String.format;
 
 /**
- * Utility for creating simple stubs for generated messages, DTOs (like {@link Event} and {@link Command}),
- * storage objects and else.
+ * Utility for creating simple stubs for generated messages, DTOs (like {@link Event} and
+ * {@link Command}), storage objects and else.
  *
  * @author Dmytro Dashenkov
  */
-public class Sample {
+public class Exemplum {
```

Comment:
The idea was to have human-readable invocations to this class methods:
```
Sample.messageOfType(Foo.class);
Sample.builderForType(Bar.class);
```
The name `Exemplum` is similar to `Example`, though not so abstract, so it could replace `Sample`.
Though the name `GeneratedMessages` says just "utility class for generating messages", not telling the main purpose of this class - generating some __sample__ non-empty data.

I'm not sure if we want to cut that descriptiveness in the name of one more utility name.